definitions/SpellPower.dbd

```diff
@@ -103,3 +95,43 @@ Field_1965009538<8>
 Field_1335777086<32>
 Field_277899929<u32>
 $noninline,relation$SpellID<32>
+
+BUILD 5.0.1.15464-5.0.1.15544
+$id$ID<32>
+PowerType<32>
+ManaCost<32>
+ManaCostPerLevel<32>
+ManaPerSecond<32>
+PowerDisplayID<32>
+AltPowerBarID<32>
+PowerCostPct
+Field_3005919490
+RequiredAuraSpellID<32>
+SpellID<32>
+
+BUILD 4.3.2.15211-4.3.4.15595
+$id$ID<32>
+ManaCost<32>
+ManaCostPerLevel<32>
+PowerType<32>
```

Comment:
At least this version has wrong field order. This column is actually cost. I fear we have a column type depending on version here as I guess you mapped the name to the only existing float.. 

Likely: last column is the unnamed float, this column is cost (but integer!) and some other one vanished.